.github/workflows/tests.yml

```diff
@@ -119,7 +119,7 @@ jobs:
 
     runs-on: ubuntu-20.04
     container:
-      image: ghcr.io/newrelic/newrelic-python-agent-ci:latest
+      image: ghcr.io/newrelic/newrelic-python-agent-ci:sha-b0ffe8bdbb28ba0579377076ad680054da8fbc28
```

Comment:
We've still got a dev sha pinned here and it's merged into main now. We'll have to fix that in a different PR.